editor/src/core/shared/github/helpers.ts

```diff
@@ -855,55 +856,115 @@ export function mergeProjectContentsTree(
   }
 }
 
-const GITHUB_REFRESH_INTERVAL_MILLISECONDS = 30_000
+const GITHUB_REFRESH_INTERVAL_MILLISECONDS = 15_000
 
-let githubPollingTimeoutId: number | undefined = undefined
+export function githubOperationContext(): GithubOperationContext {
+  // this copied here from core/shared/github/operations/index.ts due to a circular import
```

Comment:
the reason for the operation context object is easy mocking, but that ONLY works if it's in a shared "blessed" location and everyone knows that is the one place to mock

Reply:
I'm gonna try and move the const context to a different location

Reply:
https://github.com/concrete-utopia/utopia/pull/5389/commits/80b7a3555ceff01e1c43d2b231c85a662d4fb1fb